.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,32 +2,41 @@
 
 Thank you for contributing to RAPIDS Accelerator for Apache Spark!
 
-Here are some guidelines to help the review process go smoothly.
+Please read https://github.com/NVIDIA/spark-rapids/blob/HEAD/CONTRIBUTING.md#creating-a-pull-request before making this PR.
 
-1. Please write a description in this text box of the changes that are being
-   made.
+The following are the guidelines to help the review process go smoothly. Please read them carefully and fill out relevant information as much as possible.
 
-2. Please ensure that you have written units tests for the changes made/features
-   added.
-
-3. If you are closing an issue please use one of the automatic closing words as
-   noted here: https://help.github.com/articles/closing-issues-using-keywords/
+Many thanks in advance for your cooperation!
```

Comment:
```suggestion
Thank you for your cooperation!
```

Reply:
I'm not sure who wrote this originally, but this is an existing text in the current template. Even though I'm fine with either, can you explain why you suggest this?